src/test_c2s_srv.c

```diff
@@ -49,30 +49,32 @@
  * @param spds[] [] speed check array
  * @param spd_index  index used for speed check array
  * @param conn_options Connection options
+ * @param ctx The SSL context (possibly NULL)
  * @return 0 - success,
  *          >0 - error code
  *          Error codes:
  *          -1 - Listener socket creation failed
  *          -100 - timeout while waiting for client to connect to server�s ephemeral port
- * 			-errno - Other specific socket error numbers
- *			-101 - Retries exceeded while waiting for client to connect
- *			-102 - Retries exceeded while waiting for data from connected client
+ *                         -errno - Other specific socket error numbers
+ *                        -101 - Retries exceeded while waiting for client to connect
+ *                        -102 - Retries exceeded while waiting for data from connected client
```

Comment:
Is there tabs+whitespace mixing going on here? The alignment's screwy going from 57 to 58